### ISSUE TITLE
Add to SSoT

### ISSUE THREAD

**Issue by jdrew82** [abates/nautobot-app-ssot-unifi#6]
### Environment
* Nautobot version:  2.3.11
* nautobot-ssot-unifi version:  1.0.4

### Proposed Functionality
This integration should be added to the main Nautobot SSoT repo at https://github.com/nautobot/nautobot-app-ssot.

### Use Case
This would be a great integration to add into the main SSoT repo.

**Comment by jdrew82** [comment]
@abates any interest?

**Comment by abates** [comment]
@jdrew82 if you want to copy this code to the main app then go ahead. I'm going to continue to maintain this repo, however. Since I'm no longer with NTC it doesn't make a whole lot of sense for me to close this repo and lose control over the maintenance of an app that I use.

**Comment by jdrew82** [comment]
@abates Yeah that's totally fair. I dunno if you had seen that I had also created one for UniFi [here](https://github.com/jdrew82/nautobot-app-ssot-unifi) that I was going to eventually upstream. Yours is better IMO with the async usage. Would love to figure out some more integrations to push from Nautobot to UniFI, especially with the Wireless models coming in 2.4.
